python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -128,6 +128,8 @@ def load_data_to_dataframe(self, input_file: File) -> pd.DataFrame | None:
         for file in resolve_file_path_pattern(
             input_file.path,
             input_file.conn_id,
+            normalize_config=self.normalize_config,
```

Comment:
@sunank200  How adding `normalize_config=self.normalize_config` affected a CSV file?

Reply:
This should have been part of earlier PRs right? the one with pattern resolution?